scripts.json

```diff
@@ -1,5 +1,6 @@
 {
     "vendor": [
+        "https://cdn.socket.io/socket.io-1.3.4.js",
         "app/components/angular/angular.js",
```

Comment:
Hmm, thought it might. The tests aren't dependant on it, but it gets called in the run stage of `fm.player` so it would need to be mocked in all the tests?